src/data/finset/lattice.lean

```diff
@@ -103,22 +103,6 @@ sup_le (λ b hb, le_trans (h b hb) (le_sup hb))
 lemma sup_mono (h : s₁ ⊆ s₂) : s₁.sup f ≤ s₂.sup f :=
 sup_le $ assume b hb, le_sup (h hb)
 
-@[simp] lemma sup_lt_iff [is_total α (≤)] {a : α} (ha : ⊥ < a) : s.sup f < a ↔ (∀ b ∈ s, f b < a) :=
```

Comment:
What's the justification for all the moves?

Reply:
`semilattice_sup` + `is_total` gives `linear_order`. Arguably, this should be a separate PR, but it sort of happened...